umm-spec-lib/src/cmr/umm_spec/validation/platform.clj

```diff
@@ -34,24 +34,45 @@
    with the model number. So the short name is not specific enough to identify
    specific platform instances. Therefore we are adding Characteristic names and values
    so that specific aircraft tail numbers, or boat IDS, etc. can be specified with the
-   short name."
+   short name. The |-| string is also added as a delimiter between the fields. This is used
```

Comment:
The actual delimiter is the string "|-|"  while all these characters are valid it is unlikely that a user will use the combination.  Is there another combination that you would suggest?